docs/articles/documentation/test-api/assertions/assertion-api.md

```diff
@@ -0,0 +1,300 @@
+---
+layout: docs
+title: Assertion methods
+permalink: /documentation/test-api/assertions/assertion-api.html
+checked: false
+---
+# Assertion API
+
+The following assertion methods are available.
+
+* [eql](#eql)
+* [notEql](#noteql)
+* [ok](#ok)
+* [notOk](#notok)
+* [contains](#contains)
+* [notContains](#notcontains)
+* [typeOf](#typeof)
+* [notTypeOf](#nottypeof)
+* [gt](#gt)
+* [gte](#gte)
+* [lt](#lt)
+* [lte](#lte)
+* [within](#within)
+* [notWithin](#notwithin)
+
+## eql
+
+```text
+await t.expect(actual).eql(expected, message, options);
```

Comment:
In other topics we use the style `await t.expect( actual ).eql( expected, message, options );` E.g. https://github.com/DevExpress/testcafe/blob/master/docs/articles/documentation/test-api/actions/drag-element.md